airmozilla/main/templates/main/event.html

```diff
@@ -133,19 +133,24 @@ <h1 class="entry-title">
       {% endif %}
 
 
-      {% if tags %}
-        <div class="tab">
-          <input type="radio" id="tab-2" name="tab-group-1">
-          <label for="tab-2">Tags</label>
-          <div class="content">
-            <p>
-              {% for tag in tags %}
-                <a href="{{ url('main:home') }}?tag={{ tag | urlencode }}" class="tag">{{ tag }}</a>{% if not loop.last %}, {% endif %}
-              {% endfor %}
-            </p>
-          </div>
+      <div class="tab">
+        <input type="radio" id="tab-2" name="tab-group-1">
+        <label for="tab-2">Tags</label>
+        <div class="content">
+          <p>
+          {% if tags and request.user.is_active %}  
+             {% for tag in tags %}
+               <a href="{{ url('main:home') }}?tag={{ tag | urlencode }}" class="tag">{{ tag }}</a>{% if not loop.last %}, {% endif %}
+             {% endfor %}
+             -<a href="{{ url('main:event_edit', event.slug) }}" class="tag"> You can edit this here</a>
```

Comment:
Can you put this on a new line instead. No need for the `-` after the last tag. Can instead of using the word "here" let's say "You can edit the event tags"